content/articles/subscription-renewals.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: Subscription renewals
+excerpt: How to find all the relevant information about subscription renewals and how to handle failed subscription renewal payments.
+categories:
+- Account
+---
+
+# Subscription Renewals
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Depending on the billing cycle of the plan for which you signed up, your subscription will be renewed either monthly or yearly. The payment for the renewal will be charged to the card on file or, in case you have an a subscription credit, deducted from from the credit balance.
+
+The amount to be charged depends on:
+
+- The price of the plan you are subscribed to;
+- The number of extra domains you have in your account; and,
+- The number of domains that have email forwards configured.
+
+<note>
+If you want to know the exact amount you will be charged in your next renewal and when your subscription is due to renew, you can find this information on [the account page](/articles/new-account-page/#plan-breakdown).
+</note>
+
+## What happens when my subscription is renewed
+
+When the renewal of your subscription is successfully processed, we will send you an invoice and the service will be extended for the duration of your plan's billing cycle: either 1 month or 1 year.
+
+
+## What happens when the payment for the renewal fails
+
+If your subscription renewal payment fails, you will receive a dunned invoice and an email that will include details about the failure.
```

Comment:
Would it be worth linking out to an explanation of "dunned" ?

Reply:
This is a good idea

Reply:
Added! We already have an article that explains what different invoice states mean. Thanks for the suggestion.